.bazelci/presubmit.yml

```diff
@@ -89,6 +89,7 @@ coverage_validation_post_shell_commands: &coverage_validation_post_shell_command
       ; exit 1 \
       ; }
 split_coverage_postprocessing_shell_commands: &split_coverage_postprocessing_shell_commands
+  - echo "9.0.0rc3" > .bazelversion
```

Comment:
I'd prefer not to pin this here if possible.

Reply:
I'll revert this bit before landing; this was to debug the coverage duplicate actions that broke in bazel 8.5 and is another cause of red CI currently. https://github.com/bazelbuild/bazel/issues/28056